.krew.yaml

```diff
@@ -0,0 +1,41 @@
+apiVersion: krew.googlecontainertools.github.com/v1alpha2
+kind: Plugin
+metadata:
+  name: openebs
+spec:
+  version: {{ .TagName }}
+  homepage: https://openebs.io/docs/
+  shortDescription: Provides commands for OpenEBS engines.
+  description: |
+    This plugin allows you to manage storage engines provided by OpenEBS.
+  platforms:
```

Comment:
Is this consistent with the configuration present on krew already? If not do we need any additional change or will it just work? CC @niladrih

Reply:
I see that we have support for '386' arch in the older one -- https://github.com/openebs/openebsctl/blob/develop/.goreleaser.yml#L14, while this one doesn't have it. The 'description', caveat, change should be trivial.

Reply:
Alright, I double checked, and we don't list the 386 binary in the krew yaml file. We should be at par with the openebsctl repo's plugin.